pkg/console/operator/sync_v400.go

```diff
@@ -53,49 +53,63 @@ func sync_v400(co *consoleOperator, originalOperatorConfig *operatorv1.Console,
 
 	rt, rtChanged, rtErr := SyncRoute(co, operatorConfig)
 	if rtErr != nil {
-		co.SyncStatus(co.ConditionResourceSyncFailure(operatorConfig, fmt.Sprintf("%v: %s\n", "route", rtErr)))
+		msg := fmt.Sprintf("%v: %s\n", "route", rtErr)
+		fmt.Printf("incomplete sync: %v \n", msg)
```

Comment:
logrus? (here and below)

Reply:
I'm confused: is this an error or isn't it? If it's an error from the API, it seems like we should be failing. (Ideally, we'd have an error budget, though.)

Reply:
Without an error budget, its wrong to fail.  That has been explicitly stated.  But the idea of an error budget has not been fleshed out well at this point.

This may be an error (404, etc), but we should be able to tolerate it.  Its possible that the last stanza (example):

```golang
	// any error should be returned & kill the sync loop
        // if we hit this one, we may be truly failing
	if err != nil {
		logrus.Errorf("%q: %v \n", "secret", err)
		return nil, false, err
	}
	return secret, false, nil
```
Would be the correct place to truly `Failing:True`, if we had a concrete understanding of an acceptible error budget.

Reply:
Does this mean we never transition to Failing: True? What if things are completely broken and console never reconciles?